src/main/cpp/domconfigurator.cpp

```diff
@@ -113,6 +114,7 @@ IMPLEMENT_LOG4CXX_OBJECT(DOMConfigurator)
 #define STRINGSTREAM_ATTR "stringstream"
 #define CONFIG_DEBUG_ATTR "configDebug"
 #define INTERNAL_DEBUG_ATTR "debug"
+#define THREAD_TYPE_ATTR "threadConfiguration"
```

Comment:
Opinion: `THREAD_CONFIG_ATTR` seems a more reasonable name to me. `TYPE` reads like different types of threads, daemon threads vs. foreground threads vs. ..., while it's actually about their config.